k256/src/arithmetic/projective.rs

```diff
@@ -280,6 +338,32 @@ impl AddAssign<AffinePoint> for ProjectivePoint {
     }
 }
 
+impl AddAssign<&AffinePoint> for ProjectivePoint {
+    fn add_assign(&mut self, rhs: &AffinePoint) {
+        *self = ProjectivePoint::add_mixed(self, rhs);
+    }
+}
+
+impl Sum for ProjectivePoint {
+    fn sum<I: Iterator<Item = Self>>(iter: I) -> Self {
+        iter.fold(ProjectivePoint::identity(), |a, b| a + b)
+    }
+}
+
+impl<'a> Sum<&'a ProjectivePoint> for ProjectivePoint {
+    fn sum<I: Iterator<Item = &'a ProjectivePoint>>(iter: I) -> Self {
+        iter.fold(ProjectivePoint::identity(), |a, b| a + b)
+    }
+}
```

Comment:
Not sure I implemented these correctly... I suppose the implementation could also be unified

Reply:
Yes, and yes: https://github.com/zkcrypto/jubjub/blob/b7374764b47c97a126a0d171d0d0e95459fd7fdf/src/lib.rs#L168-L178